docs/components/file-upload.md

```diff
@@ -0,0 +1,213 @@
+# File Upload
+
+**KFileUpload** provides a wrapper around general `input` with `type=field` for `file upload` or `image upload`.
+ 
+<KCard>
+  <template v-slot:body>
+    <KFileUpload label="Upload File" :label-attributes="{ help: 'I use the KLabel `help` prop' }" help="Additional text." icon="close" :accept="['.yml', '.yaml', '.json', '.md', '.markdown', 'image/gif', 'image/jpg', 'image/jpeg', 'image/ico', 'image/png']" appearance="file" removable />
+  </template>
+</KCard>
+
+```html
+<KFileUpload label="Upload File" :label-attributes="{ help: 'I use the KLabel `help` prop' }" help="Additional text." icon="close" :accept="['.yml', '.yaml', '.json', '.md', '.markdown', 'image/gif', 'image/jpg', 'image/jpeg', 'image/ico', 'image/png']" appearance="file" removable />
+```
+
+## Props
+
+### appearance
```

Comment:
Instead of appearance, I think it makes more sense for this prop to be named `type` since it's actually determining the type of upload, `file` or `image`